src/fees/appliesTo.js

```diff
@@ -0,0 +1,7 @@
+import Joi from 'joi';
+
+import unitIdSchema from '../units/unitId';
+
+export default Joi.array().items(Joi.object({
+  unitId: unitIdSchema
```

Comment:
This will not always be a unit. It could be a unit, pmc, another fee, etc. I am not sure if it should be more generic. 

It is currently ```source: sourceId``` where the source could be `unitId` with a sourceId of a `shortId` value. However, we have not limited the sourceId to being a shortId type.